### PR TITLE
GraphQL and REST features missing from the docs (Cookbook for GraphQL & REST docs)

### PR DESCRIPTION
**Pull request by karensawrey**

### CODE REVIEW

File: pages/apis/graphql/graphql_cookbook.md.erb [hunk 1]
```diff
@@ -149,6 +163,49 @@ query PipelineRecentBuildLastJobQueue {
...
+          }
+        }
+      }
+    }
+  }
+}
+```
+## Get the last job of an agent
+
+To get the last job of an agent or `null`. You will need to know the UUID of the agent.
```
Comment (plaindocs) [inline pages/apis/graphql/graphql_cookbook.md.erb:188]: Is most recent more clear than last here?

**Review by plaindocs — COMMENTED** [review]
This is just changing things to sentence case and adding two recipes right? Or am I missing something?

**Review by plaindocs — APPROVED** [review]
I don't really understand `exclusive ones to the API parity doc` 

The PR title doesn't really match the content I think, so I wondered if I was missing something. Regardless, all the changes look :+1:

**Comment by karensawrey** [comment]
I meant the ones that are only present for GraphQL (hence exclusive, in my world) :D 
Merging! Thanks for the review, @plaindocs

**Pull request merged**